sdcm/nemesis.py

```diff
@@ -2262,7 +2262,8 @@ def disrupt_remove_node_then_add_node(self):
         # Repairing will result in a best effort repair due to the terminated node,
         # and as a result requires ignoring repair errors
         with DbEventsFilter(db_event=DatabaseLogEvent.RUNTIME_ERROR,
-                            line="failed to repair"):
+                            line="failed to repair",
+                            extra_time_to_expiration=120):
```

Comment:
Due to the code below this will have no effect, I think. It looks like you need to set `node` too.

Reply:
1) why do you think this won't have an effect?
2) why do you think setting the `node` attribute will help?

Reply:
it need to run for all nodes (it was already running like this before this current change).

Reply:
after chatting with @enaydanov we got to a different solution... testing...